src/mozanalysis/metrics/fenix.py

```diff
@@ -0,0 +1,95 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pyspark.sql import functions as F
+
+from mozanalysis.metrics import Metric, DataSource, agg_sum
+from mozanalysis.utils import all_
+from mozanalysis.udf import extract_search_counts
+
+
+@DataSource.from_func()
+def fenix_baseline(spark, experiment):
+    bs = spark.table('org_mozilla_fenix_baseline_parquet')
+    return bs.withColumn(
+        'client_id', bs.client_info.client_id
+    ).withColumn(
+        'b_dur', bs.metrics.timespan['glean.baseline.duration']
+    ).withColumn(
+        'duration', F.col('b_dur.value')
+    ).withColumn(
+        'first_run_date', bs.client_info.first_run_date
+    ).drop('b_dur')
+
+
+@DataSource.from_func()
+def fenix_metrics(spark, experiment):
+    met = spark.table('org_mozilla_fenix_metrics_parquet')
+    return met.withColumn('client_id', met.client_info.client_id)
+
+
+@DataSource.from_func()
+def fenix_events(spark, experiment):
+    ev = spark.table('org_mozilla_fenix_events_parquet')
+    return ev.select(
+        ev.client_info.client_id.alias('client_id'),
+        ev.submission_date_s3,
+        F.explode_outer(ev.events).alias('event')
+    ).select(
+        'event.*',
+        '*'
+    ).select(
+        '*',
+        F.explode_outer(F.col('extra'))
+    ).drop('event', 'extra')
+
+
+@Metric.from_func(fenix_baseline)
+def uri_count(met):
```

Comment:
The decorator says this is from `fenix_baseline` but the function argument suggests it's from `fenix_metrics`...